shotover-proxy/src/transforms/protect/key_management.rs

```diff
@@ -69,9 +69,8 @@ impl KeyManagerConfig {
             })),
             KeyManagerConfig::Local { kek, kek_id } => {
                 let decoded_base64 = base64::decode(&kek)?;
-                let kek =
-                    Key::from_slice(&decoded_base64).ok_or_else(|| anyhow!("Not a valid key"))?;
-                Ok(KeyManager::Local(LocalKeyManagement { kek, kek_id }))
+                let kek = Key::from_slice(&decoded_base64);
```

Comment:
The kek comes from the topology.yaml so we need some kind of validation on the size of value decoded from base64 otherwise we will hit the panic inside Key::from_slice that used to be a Result::Err.

Reply:
Oh I just realized that this logic would be better off going in the transform config validation anyway.
So we should probably leave as a panic and then raise an issue to add the relevant validation.